src/main/server/enable_ssl.sh

```diff
@@ -135,6 +136,25 @@ if [ ! -d "$INSTALL_DIRECTORY" ]; then
   exit 1
 fi
 
+freedomain(){
+  env=$(<.env)
+  hostname="ams-$RANDOM"
```

Comment:
What if same random number is generated from different instances?

Reply:
It's fixed it in the lambda function.